app/sqaws.py

```diff
@@ -130,6 +131,17 @@ def build_instance_model(pricing: PricingData, region_name: str, instance_dict:
                     nagbot_state=nagbot_state)
 
 
+# Find 'stop after' and 'terminate after' fields in an EC2 instance, regardless of formatting
+def get_stop_and_terminate_after(tags: dict) -> list:
+    stop_terminate = ['', '']
+    for key in tags:
+        if (key.lower()).startswith('stop'):
```

Comment:
It would be nice to be able to validate that the aws tag contains "after" as well. Stop and terminate alone are maybe a tad too vague.